src/website/src/pages/ec/help/faq/index.md

```diff
@@ -0,0 +1,143 @@
+
+## Where is the source code of ECL?
+
+All the library's resources are contained within a publicly-accessible [Github repository](https://github.com/ec-europa/europa-component-library).
+
+## How can I download ECL?
+
+There are several ways to get ECL resources:
+
+- [Github releases' page](https://github.com/ec-europa/europa-component-library/releases)
+- [npm @ecl](https://www.npmjs.com/org/ecl) for ECL v2
+- [Latest version CDN](https://github.com/ec-europa/europa-component-library#quick-start) for ECL v2
+- [npm @ec-europa](https://www.npmjs.com/org/ec-europa) for ECL v1
+
+## What is ECL preset? Should I use one?
+
+A preset is a set of components from a specific system (EC or EU) bundled together for distribution. Each system has a set of ready-to-use presets:
+
+- [@ecl/ec-preset-editor](https://www.npmjs.com/package/@ecl/ec-preset-editor)
+- [@ecl/ec-preset-website](https://www.npmjs.com/package/@ecl/ec-preset-website)
+- [@ecl/ec-preset-full](https://www.npmjs.com/package/@ecl/ec-preset-full)
+- [@ecl/ec-preset-legacy](https://www.npmjs.com/package/@ecl/ec-preset-legacy)
+- [@ecl/ec-preset-legacy-website](https://www.npmjs.com/package/@ecl/ec-preset-legacy-website)
+
+To access EU system presets, please use the same links as above and change `ec-` to `eu-`.
+
+When trying to decide which preset is the most appropriate for your project needs, have in mind the following:
+
+- `preset-website` is the most popular start. This preset ships all components with normalize.css and some styles applied to the document body.
+- `preset-full` is the same as `preset-website` but does not contain normalise.css and does not apply any global styles. If you are not starting the ECL implementation from scratch, but on an existing website, then this could be more appropriate preset.
+- `preset-editor` provides styles for paragraphs, lists and other generic content-related HTML elements which cannot have the necessary ECL classes applied programatically. Most frequently used in CMS or frameworks which manage content through WYSIWYG editors rather than view templates.
+
+In terms of how ECL-based websites follow up with the library's evolution when components get deprecated:
+
+- `preset-legacy-website` is an easy upgrade path to websites which have started off `preset-website` and have experienced less frequent upgrades. If any component used in `preset-website` has been deprecated and has become unavailable to a higher version, the deprecated will remain available in `preset-website-legacy` for backwards compatibility.
+- `preset-legacy` contains what has been deprecated in time.
+
+Using presets is a highly-recommended approach of implementing ECL.
+
+## Can I build/use a customised version of ECL?
+
+Short answer: yes, you can.
+
+Long answer: probably you should not. The main reason of using ECL in first place is to get all the benefits of ready-made CSS and JavaScript rules which comply with strict web governance rules.
```

Comment:
I guess that a first answer could also be to get ECL as it is and add a custom CSS on top of it.
Even if that is not optimal nor clean, that should be enough for light customizations.